packages/daimo-api/src/contract/keyRegistry.ts

```diff
@@ -96,6 +114,10 @@ export class KeyRegistry {
           addedAt: Number(log.blockNumber),
           slot,
         });
+
+        if (this.addrToDeploymentTxHash.get(addr) === undefined) {
```

Comment:
nit: `if (!this.addrToDeploymentTxHash.has(...`

Reply:
done